src/templates/zh-CHT/base.html

```diff
@@ -0,0 +1,188 @@
+</p>
+{% endblock %}
+
+{% block read_last_years_almanac %}閱讀 {{ year | int - 1 }} Web Almanac{% endblock %}
+
+{% block http_archive_link %}HTTP Archive 主頁{% endblock %}
+
+{% block page_navigation %}網頁導覽{% endblock %}
+
+{% block help_translate %}幫助翻譯{% endblock %}
+{% block translation_not_available %}英文 - 中文不能使用{% endblock %}
+{% block language_switcher %}選擇語言{% endblock %}
+{% block year_switcher %}選擇年度{% endblock %}
+
+{% block home %}主頁{% endblock %}
+{% block table_of_contents_title %}目錄{% endblock %}
+{% block contributors_title %}貢獻者{% endblock %}
+{% block methodology_title %}統計方法{% endblock %}
+
+{% block part %}部分{% endblock %}
+{% block chapter %}章{% endblock %}
+
+{% block start_exploring %}開始探索{% endblock %}
+
+{% block menu_title %}頁面目錄{% endblock %}
+{% block open_the_menu %}開啟頁面目錄{% endblock %}
+{% block close_the_menu %}關閉頁面目錄{% endblock %}
+
+{% block index_nav_title %}章節目錄{% endblock %}
+{% block index_title %}索引{% endblock %}
+{% block open_the_index %}開啟章節目錄Open the Table of Contents{% endblock %}
+{% block close_the_index %}關閉章節目錄Close the Table of Contents{% endblock %}
+
+{% block and %}和{% endblock %}
+{% block comma %}, {% endblock %}
+
+{% block open %}開啟{% endblock %}
+{% block close %}關閉{% endblock %}
+
+{% macro figure_text(metadata, id) %}圖表 {{ figure_id(metadata, id) }}.{% endmacro %}
+{% macro show_description(metadata, id) %}顯示圖表說明 {{ figure_id(metadata, id) }}{% endmacro %}
+{% macro hide_description(metadata, id) %}隱藏圖表說明 {{ figure_id(metadata, id) }}{% endmacro %}
+
+{% block written_by_before %}作者：{% endblock %}
+{% block reviewed_by_before %}審稿者：{% endblock %}
+{% block translated_by_before %}翻譯者：{% endblock %}
+{% block analysis_by_before %}分析者：{% endblock %}
+{% block written_by_after %}{% endblock %}
+{% block reviewed_by_after %}{% endblock %}
+{% block translated_by_after %}{% endblock %}
+{% block analysis_by_after %}{% endblock %}
+
+{% block author %}作者{% endblock %}
+{% block authors %}作者{% endblock %}
+
+{% macro onTwitter(twitterHandle) %}@{{twitterHandle}} Twitter{% endmacro %}
+{% macro onGitHub(gitHubHandle) %}{{gitHubHandle}} GitHub{% endmacro %}
+{% macro onLinkedIn(authorName) %}{{authorName}} LinkedIn{% endmacro %}
+{% macro website(authorName) %}{{authorName}} 網站{% endmacro %}
+
+{% macro edition(year) %}{{ year }} 版{% endmacro %}
+
+{% block footer_title %}註腳導覽{% endblock %}
+{% block copyright %}© Web Almanac. 版權聲明 <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/LICENSE">Apache 2.0</a>.{% endblock %}
+{% block accessibility_statement %}無障礙網頁聲明{% endblock %}
+
+{% block featured_chapter %}章節精選{% endblock %}
+
+{# Check if read_chapter already defined in child template as macros can't be overridden #}
+{% if not read_chapter %}
+{% macro read_chapter(chapter) %}閱讀 <span class="featured-chapter-name">{{ chapter }}</span> Chapter{% endmacro %}
```

Comment:
Should this be translated?

```suggestion
{% macro read_chapter(chapter) %}閱讀 <span class="featured-chapter-name">{{ chapter }}</span> 章節{% endmacro %}
```

Reply:
thanks. it is.